go.mod

```diff
@@ -61,7 +61,7 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0 // indirect
 	github.com/onsi/gomega v1.33.1 // indirect
 	github.com/pelletier/go-toml/v2 v2.2.2 // indirect
-	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pkg/errors v0.9.1
```

Comment:
Why is this package no longer indirect?
In general, it is no longer supported and can be replaced by the standard error.New and fmt.Errorf for now.

Reply:
fixed